Opcodes/sfont.c

```diff
@@ -990,16 +1015,17 @@ static int32_t SfInstrPlay_set(CSOUND *csound, SFIPLAY *p)
     if (UNLIKELY(index>=MAX_SFPRESET))
       return csound->InitError(csound, Str("invalid soundfont"));
     sf = &globals->sfArray[index];
-    if (*p->iskip && p->spltNum)  return OK;
```

Comment:
Doesn't taking out iskip break backwards compatibility?

Reply:
this is how the code was on Csound6